### DISCUSSION TITLE
Welcome to Koriki Discussions!

### DISCUSSION THREAD

**Discussion by eduardofilo** [Rparadise-Team/Koriki#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

**Comment by cyberbol** [comment]
Can I find anywhere any instruction how to install Super Mario Bros from NDS and necesserym BIOS on RG35XX using KORIKI firmware ?

**Comment by amiga77** [comment]
Please help as I don't know what the problem is. After uploading the Firmware - Koriki, it turns out that the volume in the games is full. Attempting to lower the volume does not work. In the Firmware options in SOUND I have no value as if it does not recognize music.

EDIT:
Ok, my mistake, because I recorded the Firmware on the SD card via BalenaEtcher, where at the end there was some error, and after uploading via RUFUS everything works as it should.

Is there any chance that Amiga ROMs will run on this firmware as they are very slow at the moment? Regards.

**Comment by paladinperry** [comment]
Hey guys. Thanks for the work on this project. I love batocera and am really looking forward to where you guys take this OS.

I know you've had some abscense and issues etc and am just wondering if you have an ETA on the next release and what to expect from it ?

Thanks again :)

**Comment by Rollingpeople79** [comment]
Buenas, soy profesor en una escuela de diseño, y estuve rediseñando el logo de Koriki por practicar un poco. Os lo paso por si queréis darle algún uso, si necesitáis alguna modificación o adaptación no dudéis en pedirlo.
[koriki-ok-color01.pdf](https://github.com/Rparadise-Team/Koriki/files/13947949/koriki-ok-color01.pdf)

**Reply by albgr1979** [reply]
Buenas!

En principio no tenemos pensada una modificación del logo pero muchas gracias por el ofrecimiento 😉

Un saludo

**Comment by hoelderlin** [comment]
Hi, how can I add the ports from Onion OS [in this link](https://github.com/OnionUI/Ports-Collection) into Koirki?
Thanks!
